pom.xml

```diff
@@ -294,20 +292,20 @@ under the License.
       <id>adangel</id>
       <name>Andreas Dangel</name>
       <email>adangel@apache.org</email>
-      <timezone>Europe/Berlin</timezone>
       <roles>
         <role>Committer</role>
       </roles>
+      <timezone>Europe/Berlin</timezone>
     </developer>
     <developer>
       <id>bdemers</id>
       <name>Brian Demers</name>
-      <organization>Sonatype</organization>
       <email>bdemers@apache.org</email>
-      <timezone>-5</timezone>
+      <organization>Sonatype</organization>
```

Comment:
That's outdated. You might want to update or remove it while at it. He's now working for Okta.

Reply:
```suggestion
```

Reply:
Well, that's completely out of scope of this PR...